debian/rules

```diff
@@ -25,6 +25,7 @@ export DEB_CFLAGS_MAINT_APPEND  = -fPIC
 override_dh_auto_configure:
 	dh_auto_configure -B$(CURDIR)/cmake-build -- \
 	-DBUILD_VERSION=$(DEB_VERSION_UPSTREAM) \
+	-DENABLE_SHARED_BSON=ON \
```

Comment:
Is linking to libbson shared necessary? I do not understand how this relates to the reported issue.

Reply:
Thank you for the explanation, I think it is OK to include as part of this commit.